client/app/dispatcher/WikiPoetryDispatcher.js

```diff
@@ -0,0 +1,13 @@
+var Dispatcher = require('flux').Dispatcher;
+
+var WikiDispatcher = new Dispatcher;
```

Comment:
Should the first letter be lower case like "wikiDispatcher" since it is an instance and not a constructor?